tsfc/driver.py

```diff
@@ -254,17 +254,25 @@ def facetarea():
     return builder.construct_kernel(kernel_name, body)
 
 
-def compile_expression_at_points(expression, points, coordinates):
+def compile_expression_at_points(expression, points, coordinates, parameters=None):
```

Comment:
We don't use it, but since `compile_ufl` now needs parameters, I guess it's fine. I might undo this if I manage simplify the argument list of `compile_ufl` during some ongoing refactorings.